ext/nnstreamer/tensor_filter/tensor_filter_llama2.cc

```diff
@@ -15,13 +15,14 @@
  * @details Pipeline example
  *          gst-launch-1.0 filesrc location=input.txt ! application/octet-stream !
  *          tensor_converter ! other/tensors,format=flexible !
- *          tensor_filter framework=llama2c model=model.bin,tokenizer.bin !
- *          filesink location=output.txt
+ *          tensor_filter framework=llama2c model=model.bin,tokenizer.bin invoke-dynamic=TRUE !
+ *          application/octet-stream ! filesink location=output.txt
```

Comment:
Is if ok to "decode" ```other/tensors``` to ```application/octet-stream``` with caps-filter?

Reply:
And.. what happens if in/out tensors are format=static?

Reply:
Oh, I missed tensor_decoder line 😅 Updated it!
This filter is for text generation, so if the tensor_filter's invoke-dynamic option is FALSE (in/out static) it thorws runtime error. 
Same with #4606